src/docs/automation-and-resiliency/netapp-backup-restore.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Restoring Backup Volumes on OpenShift
+
+slug: netapp-backup-restore
+
+description: How to request the recovery of an netapp-file-backup volume
+
+keywords: backup, restore, recovery, recover, volume, persistent, persistence
+
+page_purpose: Provides instructions for requesting the recovery of an netapp-file-backup volume
+
+audience: developer, technical lead
+
+author: Cailey Jones
+
+content_owner: Cailey Jones
+
+sort_order: 7
+---
+# Restoring Backup Volumes on OpenShift
+
+## Using `netapp-file-backup` persistent volumes
+
+The storage class `netapp-file-backup` is backed up with the standard [OCIO Backup](https://ssbc-client.gov.bc.ca/services/AppHosting/base.htm#databackup) infrastructure:
+- full backups are performed monthly
+- incremental backups are performed daily
+- backups are retained for 90 days.
+
+It's the **only** storage class that is backed up - `netapp-file-standard` and `netapp-block-standard` are not.
+
+Don't mount the `netapp-file-backup` storage class to any production workloads. It's less stable than the other two storage classes and may cause outages. You should only use `netapp-file-backup` to store backup data, preferably by mounting it to a pod controlled by a cronjob.
```

Comment:
Under the covers, the backup and standard volumes are the same. It's not accurate to say its less stable. or may cause outages. If the volume just holds regular files that don't need to have the app suspended before backup then they could use it for production. The quotas are less though, so that might pose a problem.

I agree its not something that we want to encourage.